src/components/Header/Header.jsx

```diff
@@ -1,18 +1,35 @@
 import React, { Component } from 'react';
-import './header.scss';
-import SearchBar from "../SearchBar/SearchBar";
-import styles from './header.scss'
+import SearchBar from '../SearchBar/SearchBar';
+import MovieDetails from '../MovieDetails/MovieDetails';
+import styles from './header.scss';
 
 class Header extends Component {
+  constructor() {
+    super();
+    this.state = {
+      // show: 'details',
+      show: 'search',
+    };
+  }
   render() {
     return (
       <header>
         <div className={styles.container}>
-          <SearchBar/>
+          <div className={styles.title}>netflixroulette</div>
+
+          {this.state.show === 'details' ? (
```

Comment:
You could use desctructuring here: const { show } = this.state. And let's call it showedSection instead of show